backend/smb/smb.go

```diff
@@ -107,6 +107,13 @@ Set to 0 to keep connections indefinitely.
 			Help:     "Whether the server is configured to be case-insensitive.\n\nAlways true on Windows shares.",
 			Default:  true,
 			Advanced: true,
+		}, {
+			Name: "kerberos_ccache",
+			Help: `Path to the Kerberos credential cache (krb5cc).
+
+Overrides the default KRB5CCNAME environment variable and allows each backend
```

Comment:
I think we should try to be a bit clearer here - how about

> Overrides the default KRB5CCNAME environment variable and allows this
> instance of the smb backend to use a different Kerberos cache file.

Maybe you could write why you might want to do that here too?

Also some brief documentation on the FILE/DIR/etc prefixes?

Reply:
Did some modification, let me know how it looks